lib/checkcondition.cpp

```diff
@@ -653,15 +653,14 @@ void CheckCondition::multiCondition2()
         });
 
         // parse until second condition is reached..
-        enum MULTICONDITIONTYPE { INNER, AFTER };
         const Token *tok;
 
         // Parse inner condition first and then early return condition
-        std::vector<MULTICONDITIONTYPE> types = {MULTICONDITIONTYPE::INNER};
+        std::vector<bool> conditionIsInner = {true};
```

Comment:
I don't know if something happened. But I feel that we should never ever use `vector<bool>`

Reply:
vector<bool> has the disadvantage that it is no typical STL container. But that does not matter here as it is not required that the container behaves like a STL container.